deploy/requirements.yml

```diff
@@ -39,6 +39,10 @@
   src: https://github.com/open-craft/ansible-forward-server-mail
   version: origin/master
 
+- name: consul
+  src: https://github.com/open-craft/ansible-role-consul
+  version: uman/consul
```

Comment:
Just as a reminder, this needs to be change to the master branch once the ansible-role-consul PR has been merged.

Reply:
Yes, definitely.